examples/Contact/create-contact.php

```diff
@@ -1,14 +1,14 @@
 <?php
 /**
- * @package Redbit\SimpleShop\Api
+ * @package Redbitcz\SimpleShop\Api
```

Comment:
Proč Redbitcz? Nedává ti smysl použit jenom Redbit, když název firmy je jenom Redbit? Nebo myslíš, že můžou být konflikty s jinými balíky?

Reply:
`Redbitcz` je jmenný prostor, u kterého pomocí `cz` máme určitou garanci, že tento prostor je pouze náš. Ostatně vychází to z toho, že tento prostor obsazujeme již delší dobu, viz:

- https://www.redbit.cz/
- https://github.com/redbitcz
- https://twitter.com/redbitcz
- https://packagist.org/packages/redbitcz/
- https://gitlab.com/redbitcz/

V minulosti jsme sice prefix `\Redbit` v naší interní aplikaci používali, ale to se již pozažuje za deprecated a je nahrazen `\Redbitcz`.

Reply:
Rozumím, díky